src/utils/api.enumerations.ts

```diff
@@ -11,7 +12,8 @@ const apiConstants: ApiEnumerationsConstants = yaml.load(apiConstantsFile) as Ap
 export const lookupCompanyType = (companyTypeKey: string): string => {
   // we actually use the 'company_summary' values from the yaml file to
   //  display the company type (following the ch.gov.uk templates)
-  return apiConstants.company_summary[companyTypeKey] || companyTypeKey;
+  const displayValue: string = apiConstants.company_summary[companyTypeKey] || companyTypeKey;
+  return editCompanyTypeDisplay(companyTypeKey, displayValue);
```

Comment:
we should update the api-enumerations to contain the missing values rather than try and calculate them. The api-enumerations are applied to all the other apps so the naming is consistent across the apps.
eg ukeig is a missing one, another team has added it to the api-enumerations develop branch (it should be merged into master).
I would get in touch with whoever added ukeig and see if it can be merged into master as there are probably others in develop that are not in master and would probably fix some of the missing values raised in the defect.
After that, any types that are missing should be added to api-enumerations and we can just read teh values from there. There is a confluence page for api-enumerations - https://companieshouse.atlassian.net/wiki/spaces/DEV/pages/126094319/api-enumerations